client/signup/steps/p2-site/index.jsx

```diff
@@ -465,6 +465,7 @@ class P2Site extends Component {
 					name="suggested-site"
 					value={ site ? `https://${ site }.wordpress.com` : '' }
 				/>
+				<div class="p2-site__wordpress-domain-default-container">{ this.renderDefaultSite() }</div>
```

Comment:
Would it make sense to put the container in the render function as well?

Reply:
Yes, it makes sense. I've just moved it and looks better. Thanks!